oasis3/1_create_task_json.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+path = '/lustre04/scratch/nclarke/oasis3_bids/'
+
+# JSON file content
+json_content = {
+"TaskName": "rest"
+}
+
+# Serialize JSON
+json_object = json.dumps(json_content, indent=4)
```

Comment:
Not important - we generally do 2 spaces for jsons.
```suggestion
json_object = json.dumps(json_content, indent=2)
```

Reply:
Ok, thank you!